app/assets/stylesheets/_global.scss

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  color: #222222;
+  font-family: sans-serif;
+  font-size: 1.6rem;
+  line-height: 2.5rem;
+  margin: 24px 25px 25px;
```

Comment:
Prefer to leave the body's padding alone and instead wrap our content in some kind of div.